pydy/viz/scene.py

```diff
@@ -412,9 +426,10 @@ def create_static_html(self, overwrite=False, silent=False):
         simulation_outfile.write(json.dumps(self._simulation_data_dict, indent=4,
                                 separators=(',', ': ')))
         simulation_outfile.close()
-        if not silent: print("To view the visualization, open {}".format(
-                            os.path.join(dst, 'index.html')) +
-                            " in a WebGL compliant browser.")
+        if not silent:
+            print("To view the visualization, open {}".format(
```

Comment:
I have addressed ``create_static_html()`` in my another PR. I was fixing the formatting but did it a bit over.